pdns/test-dnsdistpacketcache_cc.cc

```diff
@@ -15,6 +15,8 @@
 
 BOOST_AUTO_TEST_SUITE(test_dnsdistpacketcache_cc)
 
+static bool receivedOverUDP = true;
```

Comment:
Looks like these tests only do the `false/DoUDP` case. It might be good to add more cases (this is an issue with the existing tests already)

Reply:
I added one such unit test, but this is quite extensively tested in the regression tests.